code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -269,24 +270,28 @@
 
 		//cache for sanic speed
 		var/datum/excited_group/enemy_excited_group = enemy_tile.excited_group
-		//If we are both in an excited group, and they aren't the same, merge.
+		//If we are both in an excited group, and they aren't the same, merge your group into ours
 		//If we are both in an excited group, and you're active, share
 		//If we pass compare, and if we're not already both in a group, lets join up
 		//If we both pass compare, add to active and share
 		if(our_excited_group && enemy_excited_group)
 			if(our_excited_group != enemy_excited_group)
 				//combine groups (this also handles updating the excited_group var of all involved turfs)
 				our_excited_group.merge_groups(enemy_excited_group)
-				our_excited_group = excited_group //update our cache
```

Comment:
This isn't workable. We use our_excited_group later on, and we have no guarantee that ours is the excited group that will survive the merge